### DISCUSSION TITLE
I made a screensaver/conference eye catcher that shows solved StreetComplete quests

### DISCUSSION THREAD

**Discussion by wielandb** [streetcomplete/StreetComplete#5376] [General]
Hey all,
just wanted to share a little something I have been working on. I made a website that visualizes StreetComplete-Quests beeing solved live as little bouncing balls (a bit similar to ["show me the way"](https://osmlab.github.io/show-me-the-way/)). I personally use it as a screensaver, but I think it can also be used as an eyecatcher at a conference etc. You can try it [here](https://wielandb.github.io/fallingQuestsScreensaver/export/html5/) and look at more info [here](https://github.com/wielandb/fallingQuestsScreensaver) if you like.

![](https://user-images.githubusercontent.com/29458682/280244083-2808da7f-8c2c-43c1-bfd2-748bca1c85c5.gif)

I have to say, one thing this project showed me that I was not expecting is how much StreetComplete is actually used. I first created this project thinking my screensaver would basically be black the whole time and _occasionally_ a quest ball would drop. In reality, almost no minute passes without a lot of quests beeing solved. Makes me happy in some way that so many people share that same interest as me. 🙂

**Comment by FloEdelmann** [comment]
What a nice idea! :slightly_smiling_face:

**Comment by westnordost** [comment]
Wow, cool!
